test/stdlib/Span/StringUTF8StorageProperty.swift

```diff
@@ -0,0 +1,105 @@
+//===--- StringUTF8StorageProperty.swift ----------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2025 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+// RUN: %target-run-stdlib-swift(-enable-experimental-feature LifetimeDependence -enable-experimental-feature Span -enable-experimental-feature AddressableTypes)
+
+// REQUIRES: executable_test
+// REQUIRES: swift_feature_LifetimeDependence
+// REQUIRES: swift_feature_AddressableTypes
+// REQUIRES: swift_feature_Span
+
+import StdlibUnittest
+
+var suite = TestSuite("StringUTF8StorageProperty")
+defer { runAllTests() }
+
+suite.test("Span from Small String")
+.skip(.custom(
+  { if #available(SwiftStdlib 6.1, *) { false } else { true } },
+  reason: "Requires Swift 6.1's standard library"
+))
+.code {
+  guard #available(SwiftStdlib 6.1, *) else { return }
+
+  let s = "A small string.".utf8
```

Comment:
This will be a large string on platforms where [`_SmallString.capacity`](https://github.com/swiftlang/swift/blob/swift-6.0.3-RELEASE/stdlib/public/core/SmallString.swift#L79-L91) is 10 or 14. Also, does it matter that small strings don't store a [zero-terminator](https://github.com/swiftlang/swift/blob/swift-6.0.3-RELEASE/stdlib/public/core/SmallString.swift#L219-L225)?

The new properties are `@available(SwiftStdlib 6.2, *)`, but these new test suites are checking for `SwiftStdlib 6.1` or later.

Reply:
`Span<UInt8>` won't care about a zero terminator. C-string interop will be more specifically handled by the pitched `UTF8Span`. The availability checking in the tests will be fixed; thanks for the reminder.